src/assets/scripts/airport/AirportModel.js

```diff
@@ -180,7 +180,7 @@ export default class AirportModel {
             return;
         }
 
-        this.position = new PositionModel(currentPosition, null, magneticNorth);
+        this.position = new PositionModel(currentPosition, currentPosition, magneticNorth);
```

Comment:
To be honest, I don't think this the right solution.  It may have solved that current problem, but this creates a few more.  I still think an early exit is going to be the best solution when a `referencePosition` is not present.

Within the `PositionModel` we have:
```javascript
/**
 * @property reference_position
 * @type {PositionModel|null}
  */
this.reference_position = reference;
```
Where `reference_position` is a `PositionModel` or `null`.  Passing `currentPosition` as the second argument now introduces a third possible type for `reference_position`.  That is not a good idea because later on, you might be expecting to deal with a `PositionModel` and instead be getting a simple array.  Then things break.  

It is _sooo_ much easier for properties to be a single type.  I should mention that `PositionModel|null` is still considered the same type because a class is either an instanceof a class or null.